src/style.css

```diff
@@ -0,0 +1,113 @@
+* {
+  margin: 0;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  color: #060b35;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  font-size: 12px;
+  line-height: 18px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__product-image {
+  display: flex;
+  width: 160px;
+  height: 134px;
+  margin: auto;
+  margin-bottom: 40px;
+}
+
+.card__product-name {
+  display: block;
+  font-size: inherit;
+  font-style: normal;
+  font-weight: 500;
+  margin-bottom: 4px;
+}
+
+.card__product-code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.stars {
+  display: flex;
+  margin-bottom: 24px;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url("./images/star.svg");
+  background-size: auto;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars__star:nth-child(5) {
+  margin-right: 0;
+}
+
+.stars > .stars__star:nth-child(-n + 4) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars__reviews {
+  display: inline-block;
+  font-size: 10px;
+  margin: 0;
+  margin-left: 17px;
+  padding: 0;
+}
+
+.card__price {
+  line-height: 16px;
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__price-price_title {
+  color: #616070;
+}
+
+.card__price-price_value {
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.card__button {
+  --color: #00acdc;
+  display: flex;
+  height: 40px;
+  background: var(--color);
+  border-radius: 5px;
+  color: #fff;
+  align-items: center;
+  justify-content: center;
+  font-size: 14px;
+  font-weight: 700;
+  text-transform: uppercase;
+  box-sizing: border-box;
+  text-decoration: none;
+}
+
+.card__button:hover {
+  background: none;
+  border: 1px solid var(--color);
+  border-radius: 5px;
+  color: var(--color);
+  transition: all 0.3s;
+}
```

Comment:
The 'transition' property should be defined in the '.card__button' class instead of the ':hover' state to ensure a smooth transition when the hover state ends 🔄

Reply:
also, avoid using transition: all